src/Process/Runner/InitialTestsRunner.php

```diff
@@ -66,16 +71,35 @@ public function run(
         array $phpExtraOptions,
         bool $skipCoverage
     ): Process {
+        /** @var Process $process */
```

Comment:
```suggestion
```

The process type is documented in `ProcessBuilder` so this should not be necessary

Reply:
Updated